airbyte-cdk/bulk/toolkits/load-iceberg-parquet/src/main/kotlin/io/airbyte/cdk/load/data/iceberg/parquet/AirbyteTypeToIcebergSchema.kt

```diff
@@ -90,16 +76,90 @@ class AirbyteTypeToIcebergSchema {
             is UnknownType -> Types.StringType.get()
         }
     }
+
+    private fun convertToIcebergStruct(objectType: ObjectType): List<NestedField> {
+        return createAirbyteMetadataFields() +
+            objectType.properties.entries.map { (name, field) ->
+                if (field.nullable) {
+                    NestedField.optional(UUID.randomUUID().hashCode(), name, convert(field.type))
+                } else {
+                    NestedField.required(UUID.randomUUID().hashCode(), name, convert(field.type))
+                }
+            }
+    }
+
+    private fun createAirbyteMetadataFields(): List<NestedField> {
```

Comment:
Most of these columns are being added to the schema using the mapper architecture. if you take a look at 
```
val schema = pipeline.finalSchema.withAirbyteMeta(true).toIcebergSchema(primaryKeys)
```
In IcebergV2Writer. The `withAirbyteMeta` method is adding a bunch of em. If we want to add more like the generation id , etc. we should use the same design.